frontend/src/components/cart-page/CartSummarySection.tsx

```diff
@@ -0,0 +1,67 @@
+import { useContext } from 'react'
+import { useNavigate } from 'react-router-dom';
+
+import { ShoppingCartContext } from '../../contexts/shopping-cart-context/shoppingCartContext';
+import { Box, Button, Card, CardContent, Divider, Typography } from "@mui/material";
+
+import cartStyles from './cartStyles';
+const {
+    totalCardStyle,
+    buttonContainerStyle,
+    buttonStyle,
+} = cartStyles;
+
+const CartSummarySection = () => {
+    const navigate = useNavigate();
+    const { cartItems, userSignin } = useContext(ShoppingCartContext);
+
+    const SHIPPING_PATH = "/shipping";
+    const SIGNIN_PATH = "/user/signin";
+
+    const handleProceedToCheckout = () => {
+        userSignin ? navigate(SHIPPING_PATH) : navigate(`${SIGNIN_PATH}?redirect=${SHIPPING_PATH}`);
```

Comment:
userSignedIn?

Reply:
I left a note for you in index.tsx regarding the variables names from context, i will change many of them in a separate branch.

Reply:
yep, sorry, i missed that 👍🏼